web/src/components/storage/utils.js

```diff
@@ -19,28 +19,100 @@
  * find current contact information at www.suse.com.
  */
 
-// cspell:ignore filesize
+// cspell:ignore xbytes
 
-import { filesize } from "filesize";
+import xbytes from "xbytes";
+
+/**
+ * @typedef {Object} SizeObject
+ * @property {number} size - The "amount" of size
```

Comment:
np: So now they can be undefined, right? I would extend the docs.

Reply:
Yes. The doc update is on the way, along other changes done after reading all your comments :) Thanks!